tests/docker/BUILD

```diff
@@ -359,6 +359,43 @@ file_test(
     file = "@new_distroless_base//image:index.json",
 )
 
+file_test(
+    name = "new_alpine_linux_armv6_test2",
```

Comment:
new_alpine_linux_armv6_test_oci_layout

Reply:
And change above ones to new_alpine_linux_armv6_test_index_json and new_distroless_base_test_index_json